src/Panel/tabbedPanel.scss

```diff
@@ -1,4 +1,6 @@
-@import '../utils/theme/variables.scss';
+$tabbed-panel-tab-color: #3f4e49 !default;       // $gunmetal
```

Comment:
we're already in the context of "tabbed-panel", you can simply rename them as "tab-color" and "tab-color-active"